### PR TITLE
WAVE2-846-recreating features to add missing fields.

### PR DESCRIPTION
**Pull request by nindusaini**

### CODE REVIEW

File: sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php [hunk 1]
```diff
@@ -258,5 +258,50 @@
 <!-- How to apply Banner -->
```
Comment (gsowrey) [inline sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php]: Let's make these PHP comments, please? We don't need them in HTML.

File: sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php [hunk 2]
```diff
@@ -258,5 +258,50 @@
...
 <?php print render($content['field_fgs_banner_ref']); ?>
 
+    <!--- Supervisor's view will come here--->
 <div><?php print views_embed_view('fgs_grad_degree_supervisors', 'block'); ?></div>
 
+    <!--Completion Requirements for thesis based from story #846 -->
```
Comment (gsowrey) [inline sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php]: PHP comment, not HTML... etc.

File: sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.info [hunk 3]
```diff
@@ -28,6 +29,7 @@ dependencies[] = view_unpublished
 dependencies[] = views
 dependencies[] = views_extra_pagers
 stylesheets[all][] = css/ucgy_fgs_graduate_degree.css
+features[conditional_fields][] = node:fgs_graduate_degree
```
Comment (gsowrey) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.info]: Was this added programatically, or manually? Just wondering what it's for...

Comment (nindusaini) [inline sites/all/modules/ucalgary/ucgy_fgs/ucgy_fgs.info]: This was added by features for conditional fields we have in degree details content type.

File: sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php (hunk 2 above)
Comment (nindusaini) [inline sites/all/modules/ucalgary/ucgy_fgs/templates/node--fgs-graduate-degree.tpl.php]: Updated the comments.

**Pull request merged**